File: src/Example3Java/EX07.java
package Example3Java;

import java.util.Scanner;

public class EX07 {
    public static void main(String[] args) {
        System.out.print("Enter Text : ");
        String str = new Scanner(System.in).nextLine();
        String show = "";
        for (int i = 1; i <= str.length(); i++)
            show += str.charAt(str.length() - i);
        System.out.println("Text Reverse : " + show);
    }
}

File: src/Example3Kotlin/EX07.kt
import java.util.*

object EX07 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter Text : ")
        val str = Scanner(System.`in`).nextLine()
        var show = ""
        for (i in 1..str.length) show += str[str.length - i]
        println("Text Reverse : $show")
    }
}
